### ISSUE TITLE
feature request website

### ISSUE THREAD

**Issue by supervisitor** [picturama/picturama#60]
The slogan "Digital image organizer powered by the web" should be explained more clearly!
I already know people I have recommended this application to and they are turning away because: "... this is an online/cloud thing like Google & Co.".

**Comment by til-schneider** [comment]
Yes, you're right. I don't like the slogan either. "Digital" is unnecessary and "powered by the web" is mistakable as you say and the user doesn't care that the UI is technically run by a web browser.

The problem is, that I don't have a good alternative yet. Maybe something with "Photo organizer", "Open Source", "privacy", "fast".

Any suggestions?

**Comment by vnnv** [comment]
My suggestions is "your photos in your hands"

**Comment by supervisitor** [comment]
some more ideas, if you hold on idea to underline the technology you use:

1. next generation photo-organizer
2. a logically next step app
3. a logically next step development
4. your digital lifestyle - organized with ease
5. make use of the next-gen technology
6. make use of the advantages of next-gen technology

I favor 3. - 5. a little more

**Comment by akjoshi** [comment]
thnx for this awesome photo organizer. Suggestion based on why I used this:

1. Photo Organizer meets privacy
2. Photo Organizer for your privacy
3. Privacy First Photo Organizer
